suite-native/module-accounts/src/components/TransactionListHeader.tsx

```diff
@@ -51,28 +81,27 @@ export const TransactionListHeader = memo(
 
         const isTokenAccount = !!tokenSymbol;
 
+        // Graph is temporarily hidden also for ERC20 tokens.
+        // Will be solved in issue: https://github.com/trezor/trezor-suite/issues/7839
+        const isGraphDisplayed = accountHasTransactions && !isTestnetAccount && !isTokenAccount;
```

Comment:
I would put this condition in AccountDetailGraph itself

Reply:
We would have to put this condition to both Graph and GraphHeader. I agree that it should be the responsibility of these graph components as you suggest. But please let's wait until the graph is displayed also for eth tokens. Then the evaluation logic will be simpler and we can just check if the network is a testnet inside of the graph components.